mtools/mlaunch/mlaunch.py

```diff
@@ -1248,14 +1308,14 @@ def _construct_sharded(self):
             nextport += 1
 
 
-    def _construct_replset(self, basedir, portstart, name):
+    def _construct_replset(self, basedir, portstart, name, extra='',conf_string=[]):
```

Comment:
mutable objects as default arguments often leads to confusing issues, I would avoid.